lib/member_page.rb

```diff
@@ -96,6 +96,10 @@ def name_data
 
   def email_from(nodes)
     return if nodes.nil? || nodes.empty?
-    nodes.first.text.sub('mailto:', '')
+    email_addresses = nodes.map do |node|
+      node.text.sub('mailto:', '')
+    end
+
+    email_addresses.sort_by { |e| e.include?('parliament.gov.za') ? -1 : 1 }.join(' ; ')
```

Comment:
I fear that this method is becoming a bit too chunky here. The starting point here isn't great — this is a utility function masquerading as a method (it's only called from one place, so there's no need at all for it to take an argument) — and arose out of how we did a fairly naive migration to a more class-based version from the original everything-inlined script. So rather than continuing to layer complexity on top of that, I think it would be beneficial to first turn this into an proper instance method, so we can keep to each method only doing one thing.

I'm happy to do that part if you'd prefer to already have a better starting point, but if you're happy to do it yourself I'd suggest doing it as a separate precursor refactoring PR